src/core/client/stream/tabs/Comments/PermalinkView/PermalinkViewContainer.tsx

```diff
@@ -86,6 +87,10 @@ const PermalinkViewContainer: FunctionComponent<Props> = (props) => {
     return getURLWithCommentID(url, undefined);
   }, [pym]);
 
+  const commentVisible = !!(
+    comment && comment.status !== GQLCOMMENT_STATUS.REJECTED
```

Comment:
hmm do we not have a concept of which statuses should or should not be visible? this seems overly specific

Reply:
We do!

Right now, we have a `PUBLISHED_STATUSES` exported from `coral-server/models/comment/constants`, but we could extract those to `coral-common`.

Reply:
Unfortunately, it can't be exported to `coral-common` cause that area of the site has no access to the generated scheme types of either the server or client. To move this to `coral-common` would mean you'd need it to pull this in from one of those areas, polluting either the server code that uses it with client types or the client code that uses it with server types. 

I've settled on just using the `isPublished` already defined `src/core/client/stream/tabs/Comments/helpers/isPublished.ts`.